core/src/main/scala/com/pennsieve/managers/FileManager.scala

```diff
@@ -88,7 +90,10 @@ class FileManager(packageManager: PackageManager, organization: Organization) {
     processingState: FileProcessingState,
     size: Long = 0,
     fileChecksum: Option[FileChecksum] = None,
-    uploadedState: Option[FileState] = None
+    uploadedState: Option[FileState] = None,
+    properties: Option[Json] = None,
```

Comment:
might want to have some size validation on this object if it's coming in raw off the wire? or if you were planning on adding some schema validation later you can ignore